Sekoia.io/sekoiaio/operation_center/update_alert_status.py

```diff
@@ -0,0 +1,45 @@
+from posixpath import join as urljoin
+
+from sekoia_automation.action import Action
+import requests
+from tenacity import retry, wait_exponential, stop_after_attempt
+
+
+STATUS_UUIDS = {
+    "PENDING": "2efc4930-1442-4abb-acf2-58ba219a4fd0",
+    "ACKNOWLEDGE": "8f206505-af6d-433e-93f4-775d46dc7d0f",
+    "ONGOING": "1f2f88d5-ff5b-48bf-bbbc-00c2fff82d9f",
+    "REJECT": "4f68da89-38e0-4703-a6ab-652f02bdf24e",
+    "CLOSE": "1738b1c1-767d-489e-bada-19176621a007",
+}
+
+
+class UpdateAlertStatus(Action):
+
+    def url(self, alert_uuid: str) -> str:
+        return urljoin(self.module.configuration["base_url"], f"api/v1/sic/alerts/{alert_uuid}/workflow")
+
+    @property
+    def headers(self) -> dict:
+        api_key = self.module.configuration["api_key"]
+        return {"Authorization": f"Bearer {api_key}"}
+
+    @retry(
+        reraise=True,
+        wait=wait_exponential(max=10),
+        stop=stop_after_attempt(5),
+    )
+    def perform_request(self, alert_uuid: str, status: str):
+
+        result = requests.patch(
+            self.url(alert_uuid), headers=self.headers, json={"action_uuid": STATUS_UUIDS[status.upper()]}
+        )
+        if not result.ok:
+            self.error(f"Could not change alert {alert_uuid} status, status code: {result.status_code}")
+            result.raise_for_status()
+        return result.json()
+
+    def run(self, arguments: dict):
+        status = arguments.get("status")
```

Comment:
It would be nice to still support `action_uuid` in the code (not the manifest).

It would avoid to break existing actions configured with `action_uuid`

Reply:
oh you're right, should we add a new action to not break existing actions or just support `action_uuid` ?

Reply:
I think just supporting `action_uuid` will be fine. Having 2 separate actions will confuse the clients.

Once the module has been deploy we can run a migration to update the playbooks using `action_uuid` to rely on `status` instead.

Reply:
Okay I'll restore support for `action_uuid` in the code